src/js/api/setup-steps.js

```diff
@@ -33,7 +33,7 @@ export function loadPlaylist(_model) {
 
 function setPlaylistAttributes(model, playlist, feedData) {
     const attributes = model.attributes;
-    attributes.playlist = Playlist(playlist);
+    attributes.playlist = Playlist(playlist, model);
```

Comment:
I'm not sure why this is necessary. We've been getting by without passing in the model, here, and that has worked fine for other config values copied into sources.

While it's nice to see it on the item level, that doesn't justify all these changes. We should just copy it to sources with `formatSources` instead of adding all this new code.

Reply:
Done 👍 And it is much simpler.